src/generate.jl

```diff
@@ -64,9 +64,9 @@ function entrypoint(io::IO, pkg::AbstractString, dir)
            """
             module $pkg
 
-            greet() = print("Hello World!")
+            # module contents
 
-            end # module
+            end
```

Comment:
I actually don't think that comment is superfluous.
Because the module contents isn't indented in julia it can otherwise be confusing to have two ends at the end

Reply:
Sure, I can revert it. It's not clear to me what the "best practice" is for the closing `end`. It looks like Julia's standard library packages have a mix of `end`, `end # module` and `end # module ModuleName`.

Reply:
I think either `end # module` or `end # module ModuleName` is fine. I prefer `end # module`.

Reply:
Reverted to `end # module`